indexer-opts/meta-table.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE meta (
+    indexer_id                  VARCHAR(50)     PRIMARY KEY,
+    last_processed_block_height numeric(20, 0)  NOT NULL,
+    start_block_height          numeric(20, 0)  NOT NULL
```

Comment:
Sorry for asking to edit it one more time
Let's also add here `launched_at_timestamp`?
It will help us to debug what's going on and do we need that indexer at all

We also need something like `idexer_type`.
We now have 2 indexers writing at the same DB, it could be more. The only way to understand its nature is the id, but if we have `wind_racoon` there, we will be doomed. I want to see there `indexer-events` as well

And the order of the columns matter

```

CREATE TABLE meta (
     indexer_id                  text     PRIMARY KEY,
      idexer_type text NOT NULL,
     launched_at_timestamp  numeric(20, 0)  NOT NULL,
      start_block_height          numeric(20, 0)  NOT NULL,
     last_processed_block_height numeric(20, 0)  NOT NULL
     )
    
```
(please format it)

If we add the init function, we no longer need this, btw

Reply:
`start_block_height` actually identifies where the indexer has started.

`indexer_id` is a text I've just limited to 50 chars, but it's not necessary. So what prevents you from writing meaningful ID instead of `wind_racoon`? Set it to `indexer-events-oct13-2020-dec-15-2022`, no?

Reply:
> start_block_height actually identifies where the indexer has started.

No, I can start the indexer today, and ask it to track the blockchain from May 2021

> So what prevents you from writing meaningful ID instead of wind_racoon?

We give it as an example, so we encourage people to write there anything.
Even if you are conscious, you can just forget to write there the indexer's name and then struggle with that. It's easier just to force the user to mention the indexer type.

Reply:
> Even if you are conscious, you can just forget to write there the indexer's name and then struggle with that. It's easier just to force the user to mention the indexer type.

I totally disagree, sorry. We give tools and freedom, it's up to you how to use them. We have to advise on usage, but we don't have to control this usage.

> No, I can start the indexer today, and ask it to track the blockchain from May 2021

OK, I will add it.

Reply:
> We have to advise on usage, but we don't have to control this usage.

It's not the control, it's a help to see what's going on with your indexers.

Some more arguments: with `indexer_type`, it's easier to `group by` type and see how many indexers of each type are you running. Pavel wanted to run the indexer for each week, in this case such column could be useful.